libsyclinterface/tests/test_service.cpp

```diff
@@ -39,7 +40,7 @@ TEST(TestServicesFns, ChkDPCPPVersion)
     ASSERT_TRUE(ver.length() > 0);
 
     std::string ver_from_cmplr(ASSTR(__VERSION__));
-    std::size_t found = ver_from_cmplr.find(ver);
+    size_t found = ver_from_cmplr.find(ver);
```

Comment:
I would just use `auto found = ver_from_cmplr.find(ver)`, since `std::string::find` is documented to have return type `std::string::size_type`.